actors/miner/src/lib.rs

```diff
@@ -2515,18 +2505,6 @@ impl Actor {
         // Note: this cannot terminate pre-committed but un-proven sectors.
         // They must be allowed to expire (and deposit burnt).
 
-        {
```

Comment:
This should be safe because the _real_ limits are enforced by the "max addressed partitions/sectors" policies, but we _can't_ remove those limits (max addressed partitions/sectors) because that ends up scheduling work in cron. There's an effort to fix that here: https://github.com/filecoin-project/FIPs/pull/1035/files